bindings/cs/MongoDB.Crypt.Test/BasicTests.cs

```diff
@@ -37,13 +36,20 @@ public BasicTests(ITestOutputHelper output)
             _output = output;
         }
         
-        CryptOptions CreateOptions()
+        CryptOptions CreateOptions(bool withLocalCredentials = false)
```

Comment:
It's probably fine to always add local credentials.

Reply:
Done